# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1711561158 -3600
# Parent  c1429ee7b1bdd924064dc7604c4f3a7513c6e011
Misc: Remove account provisioner, sadly only two of the three call sites follow pref mail.provider.enabled.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -799,9 +799,9 @@ pref("mail.taskbar.lists.tasks.enabled",
 
 // Account provisioner.
 pref("mail.provider.providerList", "https://broker.thunderbird.net/provider/list");
 pref("mail.provider.suggestFromName", "https://broker.thunderbird.net/provider/suggest");
-pref("mail.provider.enabled", true);
+pref("mail.provider.enabled", false);
 pref("mail.provider.loglevel", "Warn");
 
 pref("mail.chat.enabled", true);
 // Whether to show chat notifications or not.
diff --git a/mail/components/accountcreation/content/accountSetup.xhtml b/mail/components/accountcreation/content/accountSetup.xhtml
--- a/mail/components/accountcreation/content/accountSetup.xhtml
+++ b/mail/components/accountcreation/content/accountSetup.xhtml
@@ -434,12 +434,14 @@
               class="form-icon icon-warning"
             />
           </div>
 
+          <!-- Hide the button since switching it off via mail.provider.enabled is not reliable -->
           <div class="provisioner-button-container">
             <button
               id="provisionerButton"
               type="button"
+              hidden="true"
               data-l10n-id="account-provisioner-button"
               data-l10n-attrs="accesskey"
               class="btn-link btn-link-new-email"
               onclick="openAccountProvisionerTab();"
diff --git a/mail/components/customizableui/content/panelUI.inc.xhtml b/mail/components/customizableui/content/panelUI.inc.xhtml
--- a/mail/components/customizableui/content/panelUI.inc.xhtml
+++ b/mail/components/customizableui/content/panelUI.inc.xhtml
@@ -193,12 +193,8 @@
                data-l10n-id="appmenu-new-account-panel-title"
                class="PanelUI-subView">
       <vbox id="appMenu-newViewItems"
             class="panel-subview-body">
-        <toolbarbutton id="appmenu_newCreateEmailAccountMenuItem"
-                       class="subviewbutton subviewbutton-iconic"
-                       data-l10n-id="appmenu-create-new-mail-account"
-                       oncommand="openAccountProvisionerTab();"/>
         <toolbarbutton id="appmenu_newMailAccountMenuItem"
                        class="subviewbutton subviewbutton-iconic"
                        data-l10n-id="appmenu-new-mail-account"
                        oncommand="openAccountSetupTab();"/>
